per-seat-subscriptions/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "stripe-subscription-with-fixed-price",
+  "version": "1.0.0",
+  "description": "How to create a fixed price monthly subscription and sign up form.",
```

Comment:
Do we want to update the name and description strings?

Reply:
updated, ty!